plugins/dell-kestrel/fu-dell-kestrel-hid.rs

```diff
@@ -33,3 +33,11 @@ struct FuStructDellKestrelHidFwUpdatePkg {
     dev_identifier: u8,
     fw_sz: u32be,
 }
+
+#[repr(u8)]
+#[derive(ToString)]
+enum FuDellKestrelEcRespToChunk {
```

Comment:
I think this should be called something like `FuDellKestrelHidEcChunkResponse` really

Reply:
Ok, will send fix later. Wanted to catch up 19x bus for this one.